packages/api-explorer/src/Doc.jsx

```diff
@@ -70,7 +70,7 @@ class Doc extends React.Component {
       proxyUrl: true,
     });
 
-    return fetchHar(har).then(async res => {
+    return fetchHar(har, 'ReadMe API Explorer').then(async res => {
```

Comment:
Can you make this pull in the current api explorer version from the package.json? Or maybe that doesnt matter at all actually, since we'll always know the version in production is latest.

Reply:
Yeah I'll do that since even though the version in production will always be the latest, it won't be so for old logs.